src/encoded/audit/experiment.py

```diff
@@ -3670,35 +3670,50 @@ def audit_experiment_biosample_term(value, system, excluded_types):
                 )
                 yield AuditFailure('inconsistent ontology term', detail, level='ERROR')
 
+    if value.get('assay_term_name') == 'RNA Bind-n-Seq':
+        return
+
     if 'replicates' in value:
         for rep in value['replicates']:
             if 'library' not in rep:
                 continue
 
             lib = rep['library']
-            if 'biosample' not in lib:
+            if 'biosample' not in lib and 'mixed_biosamples' not in lib:
                 detail = ('Library {} is missing biosample, expecting one of type {}'.format(
                     audit_link(path_to_text(lib['@id']), lib['@id']),
                     term_name)
                 )
                 yield AuditFailure('missing biosample', detail, level='ERROR')
                 continue
 
-            biosample = lib['biosample']
-            bs_type = biosample.get('biosample_ontology', {}).get('@id')
-            bs_name = biosample.get('biosample_ontology', {}).get('name')
             experiment_bs_type = value.get('biosample_ontology', {}).get('@id')
             experiment_bs_name = value.get('biosample_ontology', {}).get('name')
-            if bs_type != experiment_bs_type:
-                detail = ("Experiment {} contains a library {} linked to biosample "
-                    "type '{}', while experiment's biosample type is '{}'.".format(
-                        audit_link(path_to_text(value['@id']), value['@id']),
-                        audit_link(path_to_text(lib['@id']), lib['@id']),
-                        audit_link(path_to_text(bs_type), bs_type),
-                        audit_link(path_to_text(experiment_bs_type), experiment_bs_type)
+            technical_sample_type = '/biosample-types/technical_sample_NTR_0000637/'
```

Comment:
I think we should use UUID, rather than ID that could theoretically change if we get rid of NTR at some point. UUID will not change.

Reply:
I think the only way to do this is to submit it to production first with an incorrect classification (since the 'technical sample' classification is being added in this ticket) to get the UUID to use here, and patch the BiosampleType record on production after v112 release. This is what I've done in this new commit, let me know if there's a better alternative. (I can have 'dummy' values in the term_name and term_id before release also, so that v112 doesn't invalidate it)